backend/module/proxy/proxy.go

```diff
@@ -171,6 +171,36 @@ func (m *mod) RequestProxy(ctx context.Context, req *proxyv1.RequestProxyRequest
 	return proxyResponse, nil
 }
 
+func (m *mod) RequestProxyGet(ctx context.Context, req *proxyv1.RequestProxyGetRequest) (*proxyv1.RequestProxyGetResponse, error) {
+	// Validate that it's a GET.
```

Comment:
You need to validate the request permitted based on the configuration like so

https://github.com/lyft/clutch/blob/65ec44815292479e0deeab59fe2febf9c282e6e6/backend/module/proxy/proxy.go#L72-L94

Reply:
I see this is a proxy for the proxy call :)